<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondMenu</class>
 <widget class="QWidget" name="SecondMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>240</width>
     <height>210</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>240</width>
       <height>210</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1.AUTOMATIC TEST</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2.MANUAL TEST</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3.LED TEST</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4.LIGHT TEST</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>240</width>
       <height>210</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1.CHECK PAPER TEST</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2.PRINTER TEST</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3.PRINTER LINE FEED</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4.PRINTER HOT TEST</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>108</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QListWidget" name="listWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>240</width>
       <height>210</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1.ASYNC CARD TEST</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2.4442 CARD TEST</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QListWidget" name="listWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>240</width>
       <height>210</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1.GSM INCOMING CALL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2.GSM CSQ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3.GSM DIAL</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QListWidget" name="listWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>240</width>
       <height>210</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1.WIFI CONNECTION</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2.GSM CONNECTION</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3.GSM STATUS TEST</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QListWidget" name="listWidget_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>240</width>
       <height>210</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1.USB TEST</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2.TF TEST</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>241</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
